### PR TITLE
AP_Frsky_Telem: fix for passthrough telemetry stall while sending message chunks

### PR DESCRIPTION
**Pull request by yaapu**
In its present form message chunks transmission takes precedence over every other DIY packet but 0x5006 and 0x5007.
The net result is that for instance during an AUTOTUNE (very verbose) telemetry is freezed and no packets but 0x5006 and 0x5007 get a chance to be sent.
I propose a small fix that dynamically changes packet priorities based on presence/absence of pending chunks in the message queue.

Typical total bandwidth of the frsky passthrough protocol is around 40 DIY packets per seconds.

1) Current implementation no pending messages per packet type bandwidth
0x5000, 0 packets
0x5001, 1.5 packets
0x5002, 1 packet
0x5003, 1 packet
0x5004, 2 packets
0x5005, 2 packets
0x5006, 29.5 packets
0x5007, 1 packet

2) Current implementation during an autotune (message queue filled)
0x5000, 20 packets
0x5001, 0 packets
0x5002, 0 packet
0x5003, 0 packet
0x5004, 0 packets
0x5005, 0 packets
0x5006, 20.5 packets
0x5007, 1 packet

3) proposed fix no pending messages per packet type bandwidth
0x5000, 0 packets
0x5001, 1.5 packets
0x5002, 1 packet
0x5003, 1 packet
0x5004, 2 packets
0x5005, 2 packets
0x5006, 29.5 packets
0x5007, 1 packet

4) proposed fix during an autotune (message queue filled)
0x5000, 20.5 packets
0x5001, 2 packets
0x5002, 1 packet
0x5003, 1 packet
0x5004, 1.5 packets
0x5005, 1.5 packets
0x5006, 11 packets
0x5007, 1 packet

To summarize:
- the fix does not change the packet priority while the message queue is empty, see 1 and 3
- when there are pending messages 50% of the total bandwidth is assigned to messages (just like now), the remaining bandwidth is assigned 50% to attitude and 50% shared between all other packets

### CODE REVIEW

**Comment by lvale** [comment]
@peterbarker would be great to have this in.

**Comment by yaapu** [comment]
@peterbarker thanks for looking at this, I agree that the current way of dealing with priorities is "odd", my goal was to prevent stall so I left all other code untouched.

Another "odd" thing that you might not have spotted is that each messages chunk (messages are sent 4 bytes at the time) is sent 3 times and it's not clear to me why that decision was made by the library author since there's no sequence or retransmit in case of "lost" chunk.

Anyway at least now messages won't block the telemetry stream,
